### ISSUE TITLE
在使用MEND执行多模态知识编辑数据集时，后续编辑出现NAN in logits!!!的问题

### ISSUE THREAD

**Issue by QinWeijia111** [zjunlp/EasyEdit#530]
非常感谢您的团队所做的工作！
我在使用训练后的MEND模型执行关于数据集的编辑任务时
（具体代码位置：https://github.com/zjunlp/EasyEdit/blob/main/multimodal_edit.py#L552）
我训练好的MEND模型展现出了不错的效果，但是尝试了多个size大小，只要是执行的编辑的步骤超过4，就会出现`NAN in logits!!!`的问题，想请教一下这是否是MEND在执行连续编辑的时候表现不佳造成的，是否是正常的现象？

这是我的训练完成后的结果（最后的日志）：
```json
{
  "results": {
    "loss/edit_val": 0.09355704755323313,
    "loss/image_edit_val": 0.1963080149457455,
    "loss/loc_val": 0.0011949672060873127,
    "edit/acc_val": 0.765,
    "edit/log_prob_val": -0.09355704755323313,
    "edit/prob_val": 0.9670627663731575,
    "inner/acc_val": 0.766,
    "image_rephrase/acc_val": 0.597,
    "time/edit_val": 0.5838943071365357,
    "loc/acc_val": 0.980247974395752,
    "image_loc/acc_val": 0.7920896410942078,
    "loss/total_val": 0.03358714862674242,
    "loss/total_edit_val": 0.03358714862674242,
    "memory/alloc_max_val": 21436314395.648,
    "memory/res_max_val": 23581331292.16,
    "eval_time/elapsed": 1630.2304453849792,
    "eval_time/average": 1.6302304453849792
  }
}

```

以下是我在执行`edit_MEND_Blip2OPT_VQA()`函数时产生的`NAN in logits!!!`问题（只要是case_id超过3或者4就会出现这样的问题）：

![Image](https://github.com/user-attachments/assets/824f9bbe-b15b-48e8-b0f1-a0f7742fb23e)

请教以下这是否是正常的现象，`editor.edit_dataset()`是否是按照连续编辑的方式来实现，这是否是MEND方法的正常现象

**Comment by QinWeijia111** [comment]
> 您是否开了fp16量化，如果开了量化会导致这种情况

感谢您的回复！
都是默认的设置，并没有对代码进行任何的修改

**Comment by XeeKee** [comment]
那应该就是mend对于持续编辑的效果不加，您可以参考论文`MALMEN: MAssive Language Model Editing Network`
这篇论文涉及对mend的多编辑改进

**Comment by QinWeijia111** [comment]
> 那应该就是mend对于持续编辑的效果不加，您可以参考论文`MALMEN: MAssive Language Model Editing Network` 这篇论文涉及对mend的多编辑改进

好的！明白了，感谢您的回复！
